### DISCUSSION TITLE
How to check Build Queue in Vercel?

### DISCUSSION THREAD

**Discussion by surjithctly** [vercel/community#656] [Help] [answered]
Hello,

I have made a commit, and I can see this task is queued. However, I don't have any other tasks / build running. But it still shows in the queue? How do I know what's in the queue actually and how to see it?

If its an issue, how to fix it? See screenshot below. 

![CleanShot 2022-06-13 at 20 55 50@2x](https://user-images.githubusercontent.com/1884712/173389138-3b0690c6-c486-4b07-b287-42ec9afa0f46.png)

**Comment by swarnava** [comment, marked as answer]
Hi there,

Can you try now and let us know if you are still facing this issue?

**Reply by surjithctly** [reply]
It's okay now. Was it a Vercel issue?

**Reply by swarnava** [reply]
It was issue with one of our upstream providers.
